chat_exporter/parse/markdown.py

```diff
@@ -6,40 +6,46 @@
 class ParseMarkdown:
     def __init__(self, content):
         self.content = content
+        self.markdowns = []
```

Comment:
I don't feel `markdowns` is an appropriate variable name. Also, doesn't need to be established in the `__init__`, it's not past in.